README.md

```diff
@@ -173,10 +126,14 @@ Loads a single CSV file from a GitHub repository given its coordinates in an opt
 its options as follows:
 
 ````text
- {String} repoOwner - The repo owner.
- {String} repoName - The repo name.
- {String} [branchName] - [optional] The name of the remote branch to operate.
- {String} filePath - The location of the file including the path and the file name.
+{
+    repoOwner: {String} The repo owner.
+    repoName: {String} The repo name.
+    [branchName]: {String} [optional] The name of the remote branch to operate.
+    filePath: {String} The location of the file including the path and the file name.
+    octokit: {Octokit} The octokit instance holding GitHub credentials - this option is supplied automatically
```

Comment:
The definition of `octokit` is missing from the example below.

Reply:
I've changed this to {fluid.octokit} and added a clarifying comment in the "Working with GitHub credentials" section